dev/dll/XamlControlsResources.cpp

```diff
@@ -123,6 +130,18 @@ void SetDefaultStyleKeyWorker(winrt::IControlProtected const& controlProtected,
                     return L"ms-appx://" MUXCONTROLS_PACKAGE_NAME "/" MUXCONTROLSROOT_NAMESPACE_STR "/Themes/rs2_generic.xaml";
                 }
             }
+            else if (isInCBSPackage)
+            {
+                if (is19H1OrHigher)
+                {
+                    return L"ms-appx://" MUXCONTROLS_CBS_PACKAGE_NAME "/" MUXCONTROLSROOT_NAMESPACE_STR "/Themes/19h1_generic.xaml";
+                }
+                else
+                {
+                    // CBS package doesn't apply to old platforms
+                    MUX_ASSERT(false);
```

Comment:
This should be a failfast as assert doesn't fire in retail. (I can't remember what the macro is in this codebase for a retail assert or I would suggest what it is :) )

Reply:
```suggestion
                    MUX_FAIL_FAST();
```